ts_scripts/install_dependencies.py

```diff
@@ -68,8 +68,6 @@ def install_nodejs(self):
         python_path = Path(sys.executable).resolve()
         os.system(f"{self.sudo_cmd}curl -sL https://deb.nodesource.com/setup_14.x | {self.sudo_cmd}bash -")
         os.system(f"{self.sudo_cmd}apt-get install -y nodejs")
```

Comment:
The remainder of the code in here is extremely dangerous as well. First, running a shell script off the internet as root is a major security vulnerability. Second, force-upgrading `node` on a system can silently break other applications / services in the same way that force-relinking `python` can.

A little further down, `wget` is also force-upgraded, which is again potentially system-breaking.

Since we're already in here, can we remove everything that silently / forcibly overwrites system binaries?

Reply:
Hi @sharvil would you be interested in making a PR? I can make sure it works but your perspective would be invaluable